domains/pixelhu.vamtic.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "vamtic",
+    "email": "vamtic@yahoo.com"
+  },
+  "record": {
+    "A": ["35.239.22.204"]
```

Comment:
404 not found, please give a preview of your website.

Reply:
i am using a custom port just accept and only focus on syntax errors